src/test/java/seedu/address/logic/commands/book/FindBookCommandTest.java

```diff
@@ -0,0 +1,87 @@
+package seedu.address.logic.commands.book;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.book.BookAuthorContainsKeywordsPredicate;
+import seedu.address.model.book.BookNameContainsKeywordsPredicate;
+import seedu.address.model.book.BookTagContainsKeywordsPredicate;
+import seedu.address.testutil.TypicalLibTask;
+
+class FindBookCommandTest {
+
+    private Model model = new ModelManager(TypicalLibTask.getTypicalLibTask(), new UserPrefs());
+    private Model expectedModel = new ModelManager(TypicalLibTask.getTypicalLibTask(), new UserPrefs());
+
+    @BeforeEach
+    void setUp() {
+        model = new ModelManager(TypicalLibTask.getTypicalLibTask(), new UserPrefs());
+        expectedModel = new ModelManager(TypicalLibTask.getTypicalLibTask(), new UserPrefs());
+    }
+
+
+
+    @Test
+    void execute_tagPredicate_noBooksFound() {
+        String expectedMessage = String.format(Messages.MESSAGE_BOOKS_LISTED_OVERVIEW, 0);
+        //user input a tag query
+        BookTagContainsKeywordsPredicate predicate = preparePredicateTag("t/Gangster");
+        FindBookCommand findBookCommand = new FindBookCommand(predicate);
+        expectedModel.updateFilteredBookList(predicate);
+        assertCommandSuccess(findBookCommand, model, expectedMessage, expectedModel);
+        assertEquals(Collections.emptyList(), model.getFilteredBookList());
+    }
+
+    @Test
+    void execute_namePredicate_noBooksFound() {
+        String expectedMessage = String.format(Messages.MESSAGE_BOOKS_LISTED_OVERVIEW, 0);
+        //user input name query
+        BookNameContainsKeywordsPredicate predicate = preparePredicateName("n/Abandon all hope");
+        FindBookCommand findBookCommand = new FindBookCommand(predicate);
+        expectedModel.updateFilteredBookList(predicate);
+        assertCommandSuccess(findBookCommand, model, expectedMessage, expectedModel);
+        assertEquals(Collections.emptyList(), model.getFilteredBookList());
+    }
+
+    @Test
+    void execute_authorPredicate_noBooksFound() {
+        String expectedMessage = String.format(Messages.MESSAGE_BOOKS_LISTED_OVERVIEW, 0);
+        //user input author
+        BookAuthorContainsKeywordsPredicate predicate = preparePredicateAuthor("a/OOP");
+        FindBookCommand findBookCommand = new FindBookCommand(predicate);
+        expectedModel.updateFilteredBookList(predicate);
+        assertCommandSuccess(findBookCommand, model, expectedMessage, expectedModel);
+        assertEquals(Collections.emptyList(), model.getFilteredBookList());
+    }
+
```

Comment:
Good job on adding the negative test cases for Book Find command, perhaps can also add one or two successfully commands such as ```execute_validBookNameUnfilteredList_success()```  with books in the book lists🤔